python/lang/security/audit/non-literal-import.yaml

```diff
@@ -0,0 +1,14 @@
+rules:
+- id: non-literal-import
+  patterns:
+  - pattern: |
+      importlib.import_module($NAME, ...)
+  - pattern-not: |
+      importlib.import_module("...", ...)
+  message: Untrusted user input in `importlib.import_module()` function allows an attacker to load arbitrary code.
```

Comment:
What's the proper way to fix this? Is it just to use the regular import syntax?

Reply:
Updated:+1: